test/e2e/deployimage/plugin_cluster_test.go

```diff
@@ -93,7 +93,8 @@ func Run(kbc *utils.TestContext) {
 
 	By("deploying the controller-manager")
 	cmd := exec.Command("make", "deploy", "IMG="+kbc.ImageName)
-	Expect(kbc.Run(cmd)).NotTo(ContainSubstring("Warning: would violate PodSecurity"))
+	out, _ := kbc.Run(cmd)
```

Comment:
This change ignores the error here. is that intentional?

Since it goes against idioms, maybe it should be commented?

Reply:
Hi @mogsie 

IHMO: We need to have like

```
_, err = utils.Run(cmd)
		Expect(err).NotTo(HaveOccurred(), "Failed to label namespace with restricted policy")
```

Instead of `Expect(utils.Run(cmd)),` which does not make the result/output clear, we risk having an error in the string format outputted when needed.

Reply:
Reported it as #4450